src/features/hotspots/setup/HotspotSelectionScreen.tsx

```diff
@@ -0,0 +1,105 @@
+import { useNavigation } from '@react-navigation/native'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import BackScreen from '../../../components/BackScreen'
+import Box from '../../../components/Box'
+import Card from '../../../components/Card'
+import Text from '../../../components/Text'
+import TouchableHighlightBox from '../../../components/TouchableHighlightBox'
+import { HotspotSetupNavigationProp } from './hotspotSetupTypes'
+import Hotspot from '../../../assets/images/hotspot.svg'
+import RAK from '../../../assets/images/rak.svg'
+import { useColors } from '../../../theme/themeHooks'
+
+const HotspotSetupSelectionScreen = () => {
+  const { t } = useTranslation()
+  const navigation = useNavigation<HotspotSetupNavigationProp>()
+  const { purpleMain } = useColors()
+  return (
+    <BackScreen backgroundColor="primaryBackground" padding="lx">
+      <Box flex={1} />
+      <Text variant="header">{t('hotspot_setup.selection.title')}</Text>
+      <Text
+        variant="subtitle"
+        numberOfLines={2}
+        adjustsFontSizeToFit
+        marginVertical="l"
+      >
+        {t('hotspot_setup.selection.subtitle')}
+      </Text>
+
+      <Box flexDirection="row" height={191}>
+        <Card flex={1} variant="elevated" backgroundColor="white">
+          <TouchableHighlightBox
```

Comment:
I think the icon should be tinted white during the active state here

Reply:
Ok, I'll do that as well as remove react-native-vector-icons as part of this PR.